ext/hiera.yaml

```diff
@@ -2,14 +2,12 @@
 :backends:
   - yaml
 :hierarchy:
-  - defaults
-  - "%{clientcert}"
-  - "%{environment}"
-  - global
+  - "nodes/%{clientcert}"
+  - common
 
 :yaml:
 # datadir is empty here, so hiera uses its defaults:
-# - /etc/puppetlabs/code/hieradata on *nix
+# - /etc/puppetlabs/code/environments/%{environment}/hieradata on *nix
```

Comment:
This should be changed when the code actually gets changed to make that true. Until then it is still `/etc/puppetlabs/code/hieradata`